guide/how-it-works/ecash/overview.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: guide
+title: Overview
+description: A primer on ecash on bitcoin
+nav_order: 1
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/overview
+redirect_from:
+ - /guide/ecash/overview/
+main_classes: -no-top-padding -ecash-page
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/ecash.jpg
+---
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/ecash.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/ecash-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/ecash-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+Editor's notes
+
+Figma file: https://www.figma.com/file/ZwRT4mZZ4UNGtsfLz22m8M/Liquidity?node-id=0%3A1
+Figma file for channel reserve UI's: https://www.figma.com/file/6iJpftEbajA3y1ylXQxsrl/Channel-reserve
+
+-->
+
+# Ecash
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+## What is ecash?
+Ecash is a digital payment system that uses cryptographic techniques to ensure secure and private transactions. Developed by David Chaum in 1983, it operates through a mint that issues digital tokens in exchange for deposited funds. These tokens can be spent and received without revealing user identities or transaction details, thanks to blinded signatures. Users can redeem their tokens for the original funds at any time. Since no accounts or personal information are needed; the mint cannot see your balance, tokens, or transaction details, offering a highly private and efficient method for digital payments.
+
+## How ecash on bitcoin works
+Ecash on Bitcoin enhances privacy and security by using tokens for transactions. This section explains how you can create, use, receive, and convert ecash back to Bitcoin. You'll learn how protocols like Cashu and Fedimint make transactions more private and reduce onboarding friction, improving the overall user experience of transacting in Bitcoin.
```

Comment:
The paragraph uses `ecash` to refer to both the system/technology, and the tokens, which is confusing. It could be helpful to use `bitcoin tokens` when you are referring to the latter.

Reply:
`ecash tokens` also works.

It's a bit similar to our old conversations around bitcoin the network and bitcoin the currency. There's this idea that capitalization can be the differentiator, but it's so subtle that I don't think it works broadly. Better to just have some clearly different terms in general.

Reply:
as a personal preference, i like using lower case for most things.